caddy/setup/tls.go

```diff
@@ -156,6 +154,4 @@ var supportedCiphers = []uint16{
 	tls.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
 	tls.TLS_RSA_WITH_AES_256_CBC_SHA,
 	tls.TLS_RSA_WITH_AES_128_CBC_SHA,
-	tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
-	tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA,
```

Comment:
Well we don't want to remove support for them entirely. Just remove them from the default ciphers that are used.

Best bet is to make another variable called `defaultCiphers` which is the same except for these two values at the end.